src/main/Main.java

```diff
@@ -0,0 +1,141 @@
+package main;
+
+import java.util.Scanner;
+import services.*;
+
+public class Main {
```

Comment:
In coming assignments consider adding another class that tests the functionality by creating objects in the code instead of asking for manual input. This will make testing the code faster and will help to cover more cases.